packages/suite/src/support/suite/DesktopUpdater.tsx

```diff
@@ -0,0 +1,384 @@
+import React, { useCallback, useEffect, useState } from 'react';
```

Comment:
this entire file should be placed in `package/suite-desktop/src/support` (like removed VersionCheck)